e2e/vertical-full-page-map.spec.ts

```diff
@@ -1,15 +1,31 @@
 import { test, expect } from '@playwright/test';
 
+async function getAnswersVerticalResults(page) {
+  const result = page.locator('#js-answersVerticalResults div').nth(0);
+  await expect(result).toBeAttached();
+}
+
 test.describe('full page map test suite', () => {
   test.beforeEach(async ({page}) => {
     await page.goto('http://localhost:5042/locations_full_page_map');
   });
 
   test('can search and get results', async ({ page }) => {
+    const prevResponse = await page.waitForResponse(resp =>
+      resp.url().includes('https:\/\/prod-cdn\.us\.yextapis\.com\/v2\/accounts\/me\/search\/vertical\/query')
+      && resp.url().includes('queryTrigger'));
+    const prevResponseJson = await prevResponse.json();
+    const prevResultsCount = prevResponseJson.response.resultsCount;
```

Comment:
can we check the count from the ResultsCount in the UI, rather than the response, to make sure it's rendering properly?

Reply:
it will only render 20 results since the ui supports pagination so i don't think there is a ui component i can locate to check the total results count

Reply:
I thought the results count component should give you a total, like "1-20 of 72", and you could locate the "72"?